WORKSPACE

```diff
@@ -2,6 +2,22 @@ workspace(name = "distroless")
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
+# rules_distroless setup
```

Comment:
should this workspace file be in this change?

Reply:
Yea as it being Bazel lib which has the tar rule. I can bring in Bazel lib directly but I will change it once we start using rules distroless. Do you prefer I bring in Bazel lib directly?

Reply:
Oh I see, yeah this is fine.